.github/workflows/ci.yml

```diff
@@ -200,10 +200,8 @@ jobs:
         id: login
         with:
           project: firezone-staging
-      - name: Seed database
-        run: docker compose run elixir /bin/sh -c 'cd apps/domain && mix ecto.seed'
-      - name: Start docker compose in the background
-        run: docker compose up -d api web client relay gateway dns.httpbin httpbin
+      - name: Re-create database and start docker compose in the background
+        run: ./scripts/tests/wipe-and-restart-everything.sh
```

Comment:
I extracted these to a script so I can run the ecto.seed step the same way CI runs it, and so I won't forget it again. It breaks stuff silently if I forget it.